aviary/mission/flops_based/phases/detailed_takeoff_phases.py

```diff
@@ -51,10 +51,7 @@
     InitialGuessState, InitialGuessTime, PhaseBuilderBase)
 from aviary.utils.aviary_values import AviaryValues
 from aviary.variable_info.functions import setup_trajectory_params
-from aviary.variable_info.variables import Dynamic as _Dynamic
-from aviary.variable_info.variables import Mission
-
-Dynamic = _Dynamic.Mission
+from aviary.variable_info.variables import Dynamic, Mission
```

Comment:
All of these detailed takeoff phases have a "max_range" which should become "max_distance".